src/controllers/artist.js

```diff
@@ -0,0 +1,110 @@
+// exports.create = (req, res) => {
+//     res.sendStatus(201);
+// }
+// const create = (req, res) => {
+//     res.sendStatus(201);
+// }
+
+// module.exports = { create }
```

Comment:
Commented out code like this should always be removed, it just adds noise and isnt helpful